lib/language_pack/rails2.rb

```diff
@@ -48,11 +48,16 @@ def default_process_types
         "bundle exec thin start -e $RAILS_ENV -p $PORT" :
         "bundle exec ruby script/server -p $PORT"
 
-      super.merge({
+      process_types = {
         "web" => web_process,
-        "worker" => "bundle exec rake jobs:work",
         "console" => "bundle exec script/console"
-      })
+      }
+
+      if has_worker_rake_task?
```

Comment:
I wasn't sure if this is the sort of thing I should write a test for, and if so how to do that.